src/foam/u2/view/RichChoiceView.js

```diff
@@ -256,15 +262,22 @@ foam.CLASS({
                   .add(section.heading)
                 .end()
                 .start()
-                  .select(section.filtered || section.dao, function(obj) {
-                    return this.E()
-                      .start(self.rowView, { data: obj })
-                        .on('click', () => {
-                          self.fullObject_ = obj;
-                          self.data = obj;
-                          self.isOpen_ = false;
-                        })
-                      .end();
+                  .callIfElse(! section.disabled, function() {
+                    this.select(section.filtered || section.dao, function(obj) {
+                      return this.E()
+                        .start(self.rowView, { data: obj })
+                          .on('click', () => {
+                            self.fullObject_ = obj;
+                            self.data = obj;
+                            self.isOpen_ = false;
+                          })
+                        .end();
+                    });
+                  }, function() {
+                    this.select(section.filtered || section.dao, function(obj) {
+                      return this.E()
+                        .tag(self.rowView, { data: obj, disabled: true });
```

Comment:
This means that all row views must have a `disabled` property and use it, right? That's one way to do it and it's not necessarily bad, but it would be less code if we put a "disabled" style in RichChoiceView directly that applies to whatever `rowView` was given.

Maybe something like

```CSS
^disabled {
  filter: grayscale(100%) opacity(80%);
}
```

That way is less flexible, but it saves each rowView from having to account for its own disabled style. Maybe that's what we want though, I'm open to either way.

Reply:
@TW80000 with the style approach, each entry would be listed in natural order but styled differently (default provided) and still not selectable.  
Also the obj could provided a function to control disabled, which we already have with enabledAware.  By default, the function would test enabled, otherwise it is provided.

Reply:
Here we relay on developer to provide DAOs with the right data to it instead of checking each obj `enabled` property. For example,
```javascript
foam.u2.view.RichChoiceView.create({
  selectionView: { class: 'net.nanopay.auth.ui.UserSelectionView' },
  rowView: { class: 'net.nanopay.auth.ui.UserCitationView' },
  sections: [
    {
        heading: 'Contacts',
        dao: dao1
    },
    {
        disabled: true,
        heading: 'Disabled contacts',
        dao: dao2
    }
  ]
});
```